### ISSUE TITLE
Incorrect glob in the Astro parser example configuration.

### ISSUE THREAD

**Issue by virtuallyunknown** [schoero/eslint-plugin-readable-tailwind#143]
Greetings. Before anything else, I just want to quickly say thanks for developing this plugin, now that `eslint-plugin-tailwindcss` seems to be abandoned (?), I was pleasantly surprised to see an alternative pop-up, so big cheers for that.

Anyway, back to the main point, I just decided to try this for the first time and I've encountered two issues.

The first one is #112, so I think it would be a great idea if this gets mentioned in the docs. 

The second issue took me quite a while to figure out, mostly because I didn't suspect that the example configuration to be incorrect, so I went into a completely different direction trying to figure out why the plugin wasn't linting my Tailwind classes inside astro files.

The problem is very simple:

https://github.com/schoero/eslint-plugin-better-tailwindcss/blob/main/docs/parsers/astro.md?plain=1#L25

```js

export default [
  {
    files: ["*.astro"],
  }
];
```

When the `files` glob is defined in such way, I think it will only pick up `.astro` files located in the project root. The correct glob pattern should be `**/*.astro`, so:

```js

export default [
  {
    files: ["**/*.astro"],
  }
];
```

In my personal opinion, this is something very easy to overlook, since most people would probably just copy-paste the config and expect it to work.

Cheers.

**Comment by virtuallyunknown** [comment]
I think, adding an extra comment would be good enough for everyone to understand.

```js
export default [
  {
    settings: {
      "better-tailwindcss": {
        // tailwindcss 4: the path to the entry file of the css based tailwind config (eg: `src/global.css`)
        entryPoint: "src/global.css",
        // tailwindcss 3: the path to the tailwind config file (eg: `tailwind.config.js`)
        tailwindConfig: "tailwind.config.js"

        // for monorepo setups, provide a full path relative to the repo root
        // (eg: `apps/web/src/global.css`)
        // (eg: `apps/web/tailwind.config.js`)
      }
    }
  }
];
```

Wording of course can be better than what I presented here.

Cheers again!
